source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/TagContent.java

```diff
@@ -0,0 +1,86 @@
+package io.adaptivecards.renderer;
+
+import io.adaptivecards.objectmodel.BaseCardElement;
+import io.adaptivecards.renderer.inputhandler.IInputHandler;
+
+public class TagContent
+{
+
+    public TagContent(String id)
+    {
+        m_id = id;
+    }
+
+    public TagContent(BaseCardElement baseCardElement)
+    {
+        m_id = baseCardElement.GetId();
+        m_baseElement = baseCardElement;
+    }
+
```

Comment:
Should m_id just read the ID from BaseCardElement to avoid discrepancies between the ID in TagContent and the one in the BaseCardElement. If someone calls SetId and changes the Id, should that be reflected in the basecardelement.